### PR TITLE
AE-768 | ContentHeader collapse hook created

### PR DESCRIPTION
**Pull request by Shubhamkumar2023**
- New hook created for the collapse purpose
- Expose hook outside
- Ct added

### CODE REVIEW

File: src/hooks/useContentHeaderCollapse.test.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { renderHook } from "@testing-library/react";
+import { useContentHeaderCollapse } from "./useContentHeaderCollapse";
+
+describe("we should get the correct return values", () => {
```
Comment (MichalGorskiOnegini) [inline src/hooks/useContentHeaderCollapse.test.ts:4]:
```suggestion
describe("useContentHeaderCollapse", () => {
```

File: stories/FormWithStepper/FormWithStepper.stories.tsx [hunk 2]
```diff
@@ -107,4 +109,21 @@ const Template: StoryFn<Props> = args => {
   );
 };
 
+const Template: StoryFn<Props> = args => {
+  return <>{sampleFormWithStepper(() => {})}</>;
+};
+
+const Template1: StoryFn<Props> = args => {
```
Comment (MichalGorskiOnegini) [inline stories/FormWithStepper/FormWithStepper.stories.tsx:116]:
```suggestion
const TemplateWithScroll: StoryFn<Props> = args => {
```

or something similar

File: stories/FormWithStepper/FormWithStepper.stories.tsx [hunk 3]
```diff
@@ -13,6 +13,8 @@ import FormWithStepperDocumentation from "./FormWithStepper.mdx";
 import React, { useState } from "react";
 import { FormControlExample } from "../Form/Examples/FormControlExample";
 import { StepStatus } from "../../src/components/Stepper/Step";
+import { ContentHeader } from "../../src/components/Layout/ContentHeader/ContentHeader";
+import { useContentHeaderCollapse } from "../../src/hooks/useContentHeaderCollapse";
```
Comment (MichalGorskiOnegini) [inline stories/FormWithStepper/FormWithStepper.stories.tsx:18]: Can you also mention a goal of the hook somewhere in the storybook?

**Review by MichalGorskiOnegini — CHANGES_REQUESTED** [review]

File: stories/FormWithStepper/FormWithStepper.stories.tsx (hunk 2 above)
Comment (Shubhamkumar2023) [reply stories/FormWithStepper/FormWithStepper.stories.tsx]: Removed and created new story for hook

File: stories/FormWithStepper/FormWithStepper.stories.tsx (hunk 3 above)
Comment (Shubhamkumar2023) [reply stories/FormWithStepper/FormWithStepper.stories.tsx]: I have created a separate story for the hook. 
image

**Review by MichalGorskiOnegini — APPROVED** [review]

**Pull request merged**